geo/src/algorithm/sweep/tests.rs

```diff
@@ -0,0 +1,583 @@
+use super::*;
+use crate::algorithm::line_intersection::line_intersection;
+use std::iter::FromIterator;
+
+/// Helper function to compute all intersections using brute force
+fn compute_brute_force_intersections<T: GeoFloat>(
```

Comment:
This brute_force_intersection check appears slightly different from the one in the benchmarks. I don't know that it matters, but I wanted to note it in case it was oversight.